save-orchestrator/src/main/resources/application-dev.properties

```diff
@@ -1,2 +1,3 @@
 orchestrator.backend-url=http://localhost:5800/internal
-orchestrator.agent-settings.backend-url=http://host.docker.internal:5800/internal
\ No newline at end of file
+orchestrator.agent-settings.backend-url=http://host.docker.internal:5800
+orchestrator.agent-settings.orchestrator-url=http://host.docker.internal:5100
```

Comment:
please check the latest changes -- it was changed to heartbeat url

Reply:
fixed